samples/csharp_dotnetcore/58.teams-start-new-thread-in-channel/DeploymentTemplates/template-with-new-rg.json

```diff
@@ -151,42 +151,44 @@
                                 }
                             }
                         },
-                      {
-                        "apiVersion": "2017-12-01",
-                        "type": "Microsoft.BotService/botServices",
-                        "name": "[parameters('botId')]",
-                        "location": "global",
-                        "kind": "bot",
-                        "sku": {
-                          "name": "[parameters('botSku')]"
-                        },
-                        "properties": {
-                          "name": "[parameters('botId')]",
-                          "displayName": "[parameters('botId')]",
-                          "endpoint": "[variables('botEndpoint')]",
-                          "msaAppId": "[parameters('appId')]",
-                          "developerAppInsightsApplicationId": null,
-                          "developerAppInsightKey": null,
-                          "publishingCredentials": null,
-                          "storageResourceId": null
-                        },
-                        "resources": [
-                          {
-                            "name": "MsTeamsChannel",
-                            "type": "channels",
+                        {
+                            "apiVersion": "2017-12-01",
+                            "type": "Microsoft.BotService/botServices",
+                            "name": "[parameters('botId')]",
                             "location": "global",
-                            "apiVersion": "2018-07-12",
                             "kind": "bot",
+                            "sku": {
+                                "name": "[parameters('botSku')]"
+                            },
                             "properties": {
-                              "channelName": "MsTeamsChannel"
```

Comment:
Apparently this channelName property is not needed?

Reply:
This is the red line 181. You'll need to scroll down a bit to find the green line 181. The property remains, but the indentation has been corrected.